src/Illuminate/Contracts/Auth/PasswordBroker.php

```diff
@@ -34,6 +34,13 @@ interface PasswordBroker
      */
     const INVALID_TOKEN = 'passwords.token';
 
+    /**
+     * Constant representing the wait before password reset link resending.
+     *
+     * @var string
+     */
+    const RESEND_TIMEOUT = 'passwords.timeout';
```

Comment:
This is just wording, but I'm not sure about the timeout because that would mean it's timed out, but here the opposite happens. Maybe `ALREADY_SENT` or something similar.

Also after merging these changes, the laravel/laravel repo should be updated too include this in the default translation.

This should be handled somehow, so the users don't see `passwords.timeout` after update. (I don't have an idea for this, sorry)

Reply:
We have a token's lifetime limited from above by the "EXPIRED" variable.
I propose to make a limited from below restriction by disabling recreation (but not deletion).
Recreation "TIMEOUT" is the best name I could devise.
Hope someone comes up with a better one.

Reply:
The token lifetime is limited by the expire**S** variable. Like the token expires in 10 minutes.

Sadly for this I have no good idea.

P.S. I'm not a native English speaker so take my comments about the wording with a grain of salt.

Reply:
How about `SEND_THROTTLED` or `RESEND_THROTTLED` or `BROKER_THROTTLED` for the const name? And `passwords.throttled` as the translation key?

The password broker throttled the send request. Or something similar.